### PR TITLE
Add pie and line chart tabs to query result preview

### PR DESCRIPTION
**Pull request by eeriksp**

### CODE REVIEW

File: explorer/views/utils.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from django.db import DatabaseError
 
 from explorer import app_settings
+from explorer.chart import get_pie_chart, get_line_chart
```
Comment (marksweb) [inline explorer/views/utils.py:5]: Lets rename the module as it already has 2 types of chart;

```suggestion
from explorer.charts import get_pie_chart, get_line_chart
```

**Review by marksweb — CHANGES_REQUESTED** [review]

**Comment by eeriksp** [comment]
@marksweb, thank you for amendments, they have all been incorporated now.
Anything else you would like to change?

File: docs/features.rst [hunk 2]
```diff
@@ -120,6 +120,22 @@ Pivot Table
 - Hit the link icon on the top right to get a URL to recreate the
   exact pivot setup to share with colleagues.
 
+Displaying query results as charts
+----------------------------------
+
+If the results table adheres to a certain format, the results can be displayed as a pie chart or a line chart.
+
+To enable this feature, set `` `` setting to ``True`` and install the plotting libraries ``matplotlib`` and ``seaborn`` with
```
Comment (marksweb) [inline docs/features.rst:128]: Looks like this needs the actual setting name.

```suggestion
To enable this feature, set ``EXPLORER_CHARTS_ENABLED`` setting to ``True`` and install the plotting libraries ``matplotlib`` and ``seaborn`` with
```

**Review by marksweb — CHANGES_REQUESTED** [review]

**Comment by marksweb** [comment]
@eeriksp Just realised, this needs the `setup.py` updating to handle the optional dependencies added here.

There's a good example here; https://github.com/anymail/django-anymail/blob/main/setup.py#L62

At the moment we've got;

```
extras_require={
        "xls": [
            'xlsxwriter>=1.2.1'
        ]
    },
```

So this could go in with the feature name;

```
extras_require={
        "graphs": [
            'matplotlib<4',
            'seaborn<0.12'
        ]
        "xls": [
            'xlsxwriter>=1.2.1'
        ]
    },
```

**Pull request merged**